### DISCUSSION TITLE
How to create Iceberg table with parquet format and snappy compression files.

### DISCUSSION THREAD

**Discussion by perceptron01** [trinodb/trino#23338] [Q&A]
I wan to create a Iceberg table with parquet format and snappy compression files.
According to the [Document](https://trino.io/docs/current/connector/iceberg.html#sorted-tables), we can specify the file format when creating a iceberg table as follows.
```sql
CREATE TABLE example_table (
    c1 INTEGER
)
WITH (
    format = 'PARQUET'
);
```
But it was not clear from the documentation how to specify that the file should be SNAPPY compressed.
As https://github.com/trinodb/trino/discussions/15037 says, I tried `set session` command, but this doesn't work.
```
trino:default> SET SESSION iceberg.compression_codec = 'SNAPPY';
            -> CREATE TABLE example_table (
            ->     c1 INTEGER
            -> )
            -> WITH (
            ->     format = 'PARQUET'
            -> );
```
```
trino:default> SELECT * FROM default."example_table$properties";
               key               |             value              
---------------------------------+--------------------------------
 created-at                      | 2024-MM-DDTHH:MM:SS.000000000Z 
 write.format.default            | PARQUET                        
 write.parquet.compression-codec | zstd  
```
Is there a way to specify compression codec for iceberg tables in Trino?

**Comment by ebyhr** [comment]
It's related to https://github.com/trinodb/trino/issues/20401

The session property affects only when writing data files.
